### PR TITLE
Copr conglomerator.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/conglomerators/copr/copr.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                      if m['topic'].endswith('copr.build.start')])
+
+        rebuild_predicate = 'rebuilds' if starts > 1 else 'rebuild'
+        copr_predicate = 'coprs' if len(coprs) > 1 else 'copr'
+
+        agents = self.list_to_series(agents)
+        coprs = self.list_to_series(coprs)
+
+        subtitle = '{agents} kicked off {starts} {rebuild_predicate} ' + \
+            'of the {coprs} {copr_predicate}'
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/conglomerators/copr/copr.py]: of vs in ?

**Comment by pypingou** [comment]
On question, otherwise :+1:

File: fedmsg_meta_fedora_infrastructure/conglomerators/copr/copr.py (hunk 1 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/conglomerators/copr/copr.py]: I think *of*?  I dunno.  I think *of*.

**Comment by ralphbean** [comment]
Thanks again @pypingou!

File: fedmsg_meta_fedora_infrastructure/conglomerators/copr/copr.py (hunk 1 above)
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/conglomerators/copr/copr.py]: I was thinking: `` foo kicked of 2 rebuilds in the bar copr``

Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/conglomerators/copr/copr.py]: Yeah, I thought ``foo kicked off 2 rebuilds of the bar copr`` sounded more natural.  Let's get a third opinion from someone else.

**Pull request merged**